docs/retrieving-stats.rst

```diff
@@ -9,13 +9,31 @@ First, when running Locust with the web UI, you can retrieve CSV files under the
 Secondly, you can run Locust with a flag which will periodically save the CSV file. This is particularly useful
 if you plan on running Locust in an automated way with the ``--no-web`` flag::
 
-    locust -f locust_files/my_locust_file.py --csv=foobar --no-web -n10 -t10m
+    locust -f examples/basic.py --csv=example --no-web -n10 -t10m
 
```

Comment:
is `example` the full name of the csv file?  Might want to name it `example.csv`

Reply:
it gets prepended with the type of file, so providing `example` results in `example_distribution.csv` and `example_requests.csv`

Reply:
gotcha.  that's confusing.  maybe note that in the docs?

Reply:
I added clarification there